extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -40,3 +40,13 @@ amp-sticky-ad {
 .-amp-sticky-ad-layout > amp-ad {
   display: block;
 }
+
+.-amp-sticky-ad-close-button {
+  position: absolute;
```

Comment:
Wouldn't this make button invisible given the `overflow:hidden` on the fixed `amp-sticky-ad`?